views/term_table.erb

```diff
@@ -1,3 +1,18 @@
+<div class="page-section page-section--green page-section--no-padding">
+    <div class="container text-center">
+        <ul class="house-tabs">
+          <% @country[:legislatures].each do |house| %>
+            <li>
+                <% latest_term = house[:legislative_periods].sort_by{ |term| term[:start_date] }.last %>
```

Comment:
These are already pre-sorted (though in most-recent-first order), so you just need a `house[:legislative_periods].first` there (which also means you can probably inline it rather than setting up a `latest_term`).